apollo-service/values.yaml

```
@@ -43,6 +43,7 @@ configService:
   imagePullSecrets: []
   service:
     fullNameOverride: ""
+    annotations: { }
```

Should we remove the extra space?
```suggestion
    annotations: {}
```

---

I removed all the extra spaces